src/server.py

```diff
@@ -0,0 +1,71 @@
+"""Server for Echo server assignment."""
+# encoding: utf-8
+from __future__ import print_function
+import socket
+
+
+def server():
+    serv = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
+    address = ('127.0.0.1', 5002)
+    serv.bind(address)
+
+    serv.listen(1)
+
+    while True:
+        conn, addr = serv.accept()
+        try:
+            req_string = b''
+            buffer_length = 10
+            while req_string[-4:] != b"\r\n\r\n":
```

Comment:
You could also use the `.endswith` message of both unicode strings and bytestrings.